### DISCUSSION TITLE
Accessibility to S21 at Emura 3

### DISCUSSION THREAD

**Discussion by mpmario** [revk/xESP32-Faikout#525] [Q&A]
Hello everyone,
I'm experiencing a technical issue with my Daikin Emura 3 devices and hope someone here has experience with this.
My situation:
I own six relatively new Emura 3 devices (models FTXJ20AW and FTXJ42AW)
In the electrical connection box, I found an S21 cable connected to a socket (see picture)
When measuring the voltage on the pins of this socket, there's no reading
Interestingly, the Onecta App continues to function even when I disconnect the S21 cable
My questions:
Am I checking the correct socket, or do I need to open the device further?
Has anyone had similar experiences or suggestions for a solution?
Should there be a measurable voltage on this socket at all?
I appreciate any insights. Thank you in advance for your help!
![IMG_0383](https://github.com/user-attachments/assets/11097fd1-d0e2-4394-8bea-395e85e96dd0)
![IMG_0384](https://github.com/user-attachments/assets/32c12cb8-0971-411f-ba3b-93db38c7dcd9)

Thank you!
Mario

**Comment by FloHomi** [comment]
Hi this is the connector for the motorised front cover.
I had some troubles getting into there, but now I know how to do it :)
you have to remove the whole outer shell (one piece)
on the bottom of the "air out" there are two screws. roughtly where the circles are. 
there are clips on the top, where the white cover touches the wall. (see manual)


![376344442-11097fd1-d0e2-4394-8bea-395e85e96dd0](https://github.com/user-attachments/assets/25dcc462-5425-42e3-bba5-06706c5c140f)

The S21 connector is on the right, underneath an extra metal cover.

![1729145172260](https://github.com/user-attachments/assets/d0e3b3ea-a526-400d-8fcc-2387429cd164)

**Reply by mpmario** [reply]
Thank you very much for the hints!!!

It was difficult to remove the front cover as the unit was very high in the room and there was not much space above it, but I managed it.

For reference:
Here you need to remove the cover in front of 2 screws. They are very hard to get out, so use a screwdriver with a lot of force.
![IMG_0512](https://github.com/user-attachments/assets/278bea46-a5ef-4f1d-b4c6-0e3916b69f76)
![IMG_0513](https://github.com/user-attachments/assets/c773568c-a665-4507-90b3-0bab6bddeb0b)

Then remove the complete front cover. Carefully lift the upper rear part to release it from the 3 hooks. 
![IMG_0514](https://github.com/user-attachments/assets/fe55dc69-9f97-4680-ab62-2a49ba11e29f)

Then the port is accessible.
![IMG_0517](https://github.com/user-attachments/assets/0bd23a97-5328-4e5a-b222-481b58b5821a)

More information in the installer reference manual.
